docs/basics/101-111-run3.rst

```diff
@@ -0,0 +1,140 @@
+Content-protection, powered by Git-annex [1]
+--------------------------------------------
+
+**Oh crap!** A "permission denied" error? Confused about what we might have
+done wrong, we raise our hand to ask the instructor for help.
+Knowlingly, she smiles, and tells you about how DataLad protects content given
+to it:
+
+"Content in your DataLad dataset is protected by :term:`Git-annex` from
+accidental changes" our instructor begins.
+
+"Wait!" we interrupt. "First of, that wasn't accidental. And second, I was told this
+course does not have ``Git-annex-101`` as a prerequisite?"
+
+"Yes, hear me out" she says. "I promise you two different solutions at
+the end of this explanation, and the concept behind this is quite relevant".
+
+DataLad usually gives content to :term:`Git-annex` to store and track.
+Git-annex, let's just say, takes this task *really* seriously. One of it's
+features that you have just experienced is that it *locks* content.
+
+If files are *locked down*, their content can't be modified. In principle,
+thats not a bad thing: It could be the presidents nuclear launch code in that file,
```

Comment:
In this day and age, we should find another example, or this book will make it into the post-apocalyptic section on amazon... ;-)